spec/classes/init_spec.rb

```diff
@@ -3285,6 +3285,63 @@
       end
     end
   end
+  
+  describe 'with ssh_key_export parameter specified' do
+    context 'as a non-boolean or non-string' do
+    let(:params) { { :ssh_key_export => ['not_a_boolean','or_a_string'] } }
+
+      it 'should fail' do
+        expect {
+          should contain_class('ssh')
+        }.to raise_error(Puppet::Error)
```

Comment:
please put the actual error message in here. A way to find out is to put 

```ruby
}.to raise_error(Puppet::Error,/asdf/)
```

and learn from the spec failure